src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -43,14 +47,14 @@
         Map<String, Object> transformedCaseData = new HashMap<>();
 
         mapFullNameToFirstAndLast(OcrFieldName.APPLICANT_FULL_NAME, "applicantFMName", "applicantLName",
-            ocrDataFields, transformedCaseData);
+                ocrDataFields, transformedCaseData);
```

Comment:
I think we actually said we're going to have 4 spaces for continuation afair. @yarpo ?